src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,47 @@
 
 class Animal {
   // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+  }
```

Comment:
```suggestion
  constructor(name) {
    this.name = name;
    this.health = 100;
    Animal.addAnimal(this);
  }
```
I think you could add animals here, in the base constructor. It excludes the duplication of this logic in other constructions because of super.